easybuild/easyconfigs/y/Yambo/Yambo-4.0.2-rev.90-intel-2015b-QuantumESPRESSO-5.2.1.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'MakeCp'
+
+name = 'Yambo'
+version = '4.0.2-rev.90'
+qe = 'QuantumESPRESSO'
+qever = '5.2.1'
+versionsuffix = '-%s-%s' % (qe, qever)
+
+homepage = 'http://www.yambo-code.org'
+description = """Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.
+ Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://qe-forge.org/gf/download/frsrelease/201/870/']
+checksums = ['03d5ffc655eb4c03db4e264606117c86']
+
+dependencies = [
+    ('netCDF-Fortran', '4.4.2'),
+    (qe, qever),
+]
+
+with_configure = 'True'
+
+configopts = 'CPPFLAGS="" FCFLAGS="-nofor_main" '
+#configopts = 'CC=$CC_SEQ MPICC=$CC CPPFLAGS="" FCFLAGS="-nofor_main" '
+configopts += '--with-blas-libs="$LIBBLAS" '
+configopts += '--with-lapack-libs="$LIBLAPACK" '
+configopts += '--with-fft-libs="$LIBFFT" '
+configopts += '--with-iotk-path="$EBROOTQUANTUMESPRESSO/espresso-%s/S3DE/iotk" ' % qever
+configopts += '--enable-netcdf-hdf5 '
+configopts += '--with-netcdf-includedir="$EBROOTNETCDFMINFORTRAN/include" '
+configopts += '--with-netcdf-libdir="$EBROOTNETCDFMINFORTRAN/lib" '
+
+buildopts = 'yambo interfaces ypp'
+
+parallel = 1
+
+files_to_copy = [(['bin/*'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/'+prog for prog in ['a2y', 'p2y', 'yambo', 'ypp' ]],
+    'dirs': []
+}
+
+modtclfooter = "remove-path	PATH	$::env(EBROOTQUANTUMESPRESSO)/espresso-%s/YAMBO/bin" % qever
```

Comment:
@boegel  It is not necessary to remove the YAMBO/bin from the path, because the first appearance will be used.  I just do not like if a program is twice in the path.
And I do not have any idea how to do in lua.